core/include/prometheus/detail/utils.h

```diff
@@ -1,4 +1,4 @@
-#pragma onece
```

Comment:
good catch. don't know why I did not notice that.

Reply:
@gjasny  Me too.